lib/elasticity/index_config.rb

```diff
@@ -1,9 +1,11 @@
 module Elasticity
   class IndexConfig
-    ATTRS = [:index_base_name, :document_type, :mapping, :strategy].freeze
+    ATTRS = [:index_base_name, :document_type, :mapping, :strategy, :subclasses].freeze
+
     attr_accessor *ATTRS
 
-    def initialize(elasticity_config)
+    def initialize(elasticity_config, default_document_type)
```

Comment:
Why add another parameter to IndexConfig if the document type can be set directly into `elasticity_config` ?